src/libraries/System.Net.Sockets/tests/FunctionalTests/NetworkStreamTest.cs

```diff
@@ -271,36 +291,6 @@ public async Task DataAvailable_ReturnsFalseOrTrueAppropriately()
             });
         }
 
-        [Theory]
-        [InlineData(false)]
-        [InlineData(true)]
-        public async Task DisposedClosed_MembersThrowObjectDisposedException(bool close)
-        {
-            await RunWithConnectedNetworkStreamsAsync((server, _) =>
-            {
-                if (close) server.Close();
-                else server.Dispose();
-
-                Assert.Throws<ObjectDisposedException>(() => server.DataAvailable);
```

Comment:
Might be worth keeping this just for DataAvailable

Reply:
Ok